includes/class-zoninator-api-controller.php

```diff
@@ -680,6 +683,17 @@ public function _params_for_search_posts() {
 //		);
 	}
 
+	public function _filter_zone_properties( $zone ) {
+		$data = $zone->to_array();
```

Comment:
Are we sure the parameters we use are present? If not we should check and initialize as sensible defaults (by something like `empty( $data['name'] ) ? '' : $data['name'];`.

Reply:
Yes, as these are fresh zone objects fetched from the DB - not the ones from the request.